src/logic/calculate.js

```diff
@@ -26,14 +17,14 @@ export default function calculate(obj, buttonName) {
     if (buttonName === '0' && obj.next === '0') {
       return {};
     }
-    // If there is an operation, update next
```

Comment:
[Optional] I don't think it was necessary to remove these comments since they're meaningful and help to understand the code better.